client/src/index.jsx

```diff
@@ -80,7 +80,7 @@ class App extends React.Component {
         </div>
         <Overview starValue={this.state.starValue} productId={this.state.id} reviewsNumber={this.state.reviewLength} getProductName={this.getProductName.bind(this)} />
         <RelatedLists productId={this.state.id} starValue={this.state.starValue} selectProduct={this.selectProduct} />
-        <QuestionsAnswersWithClickTracking id={this.state.id} />
+        <QuestionsAnswersWithClickTracking id={1} />
```

Comment:
This is hardcoded because the rest of the client was operating with a different set of ids than the ones in my database